ignite/services/network/profile.go

```diff
@@ -53,6 +53,33 @@ func (n Network) Coordinator(ctx context.Context, address string) (networktypes.
 	return networktypes.ToCoordinator(resCoord.Coordinator), nil
 }
 
+// SetValidatorDescription set a validator profile.
+func (n Network) SetValidatorDescription(ctx context.Context, validator profiletypes.Validator) error {
+	n.ev.Send(events.New(events.StatusOngoing, "Setting validator description"))
+
+	addr, err := n.account.Address(networktypes.SPN)
+	if err != nil {
+		return err
+	}
+
+	message := profiletypes.NewMsgUpdateValidatorDescription(
+		addr,
+		validator.Description.Identity,
+		validator.Description.Moniker,
+		validator.Description.Website,
+		validator.Description.SecurityContact,
+		validator.Description.Details,
```

Comment:
`Moniker` and `SecurityContact` are not updatable with the `validator set` , is this OK?

Reply:
I work on this PR primarily for the `identity` field as questions were raised for this on Discord but the complexity to add those is not high, I will do it

Reply:
> Got it. I suppose that passing an empty field doesn't erase the existing field in the store, right?

No, empty is ignored

Reply:
Just adding `SecurityContact `, `Moniker` will not have usage here